docs/release-notes.md

```diff
@@ -10,6 +10,90 @@ toc_max_heading_level: 2
+
+#### Changes
+- Clicking on the Project Name in the Projects page now takes you to that project's Findings page. Click the gear icon at the end of the Project's row to go to the project's Settings page.
+- Semgrep App detects additional environment variables depending on your provider. This simplifies the creation and committing of the configuration file when adding a new Project (repository) in Semgrep App.
+- UI and UX improvements to **Scan new project** workflow.
+
+### Semgrep CLI
+
+These release notes include upgrades for all versions ranging between 0.102.0 and 0.107.0.
+
+#### Additions
+
+- Semgrep in CI:
+  - Fail-open support: Added `--suppress-errors` and `--no-suppress-errors` (the default is `--no-suppress-errors`).
+  - Support for podman environments!
+  - Semgrep in CI does not block builds on triage ignored issues.
+  - The timeout for Git commands Semgrep runs is now configurable. To configure the timeout, set the `SEMGREP_GIT_COMMAND_TIMEOUT` environment variable. The time unit used as value for this key is in seconds. The default value is 300.
+  - The `SEMGREP_GHA_MIN_FETCH_DEPTH` environment variable lets you set how many commits `semgrep ci` fetches from the remote at the minimum when calculating the merge-base in GitHub Actions. Having more commits available helps Semgrep determine what changes came from the current pull request, fixing issues where Semgrep would otherwise report findings that were not touched in a given pull request. This value is set to 0 by default. (Issue [#5664](https://github.com/returntocorp/semgrep/pull/5664))
+
+- Extract mode:
+  - New Semgrep CLI experimental extract mode. This mode runs a Semgrep rule on a codebase and extracts code from matches, treating it as a different language. This allows you to supplement an existing set of rules, for example, for JavaScript, by writing additional rules to find JavaScript in files of a different language than JavaScript. For example, JavaScript code in HTML or template files. While this is somewhat possible with `metavariable-pattern`, this reduces the work from an M \* N problem to an M \+ N. To know more about extract mode, see [Extract mode](experiments/extract-mode.md) documentation.
+  - Extract mode now has a concatenation reduction (`concat`). Disjoint snippets within a file can be treated as one unified file. 
+  - You can use extract mode to scan for generic languages (use value `generic` in `dest-language`).
+
+- Taint mode:
+  - Add experimental support for _taint labels_, which is an ability to attach labels to different kinds of taint. Both sources and sinks can restrict what labels are present in the data that passes through them in order to apply. This allows you to write more complex taint rules that previously required unappealing workarounds. Taint labels are also helpful for writing certain classes of typestate analyses (for example, check that a file descriptor is not used after being closed).
```

Comment:
As a user, how do I use it or enable it?

Reply:
Not sure here, would have to research with @IagoAbal. Please, can you give us some references on how to use taint labels? This can be a part of different PR and we do not have to put it in release notes, to rule-syntax document, please https://semgrep.dev/docs/writing-rules/rule-syntax/ 🙏